pkg/crc/machine/machine.go

```diff
@@ -3,19 +3,17 @@ package machine
 import (
```

Comment:
The commit log is lacking some explanations why we want to do this.

Reply:
This is WIP to have it tested properly. If OK, then I'll submit the PR for merging.